tests/regression/mmrotate.yml

```diff
@@ -0,0 +1,50 @@
+globals:
```

Comment:
should we update mmrotate support in `tools/regression_test.py` accordingly?

Reply:
I am not so sure what should I add.
Actually, I can not test the accuracy since the dataset require update the result to the server.

Reply:
I think we should support it. We can test model conversion at least. 
What are the concerns?

Reply:
Agree. Conversion could be tested.

Reply:
All conversions tested success on my side.